ci-operator/step-registry/azure/provision/disk-encryption-set/azure-provision-disk-encryption-set-ref.yaml

```diff
@@ -5,6 +5,10 @@ ref:
     name: "4.12"
     tag: upi-installer
   commands: azure-provision-disk-encryption-set-commands.sh
+  env:
+  - name: SET_DIFFERENT_DES_FOR_COMPUTE
```

Comment:
we do not  need this ENV any more, right?

Reply:
yes, removed.